.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -2,7 +2,7 @@
 name: Bug report
 about: Report a problem in Nvim
 title: ''
-labels: bug
+labels: '0.kind: bug'
```

Comment:
Where is this kind of stuff documented?

Reply:
I mentioned it on the chat it's a proposition of mine feel free to disagree ^^
 The goal is via this prefix to bring some order/self-document our 120 labels so that users can better filter issues/PRs. Also it makes the display of tags more consistant (kind arrives first). See https://github.com/NixOS/nixpkgs/issues for an example. I used to find it verbose but I better see the value now.

Reply:
Oh, I somehow assumed it was some kind of special syntax, not just a prefix to the label. In that case.. go on. I like the idea. :)